login/scss/detail.scss

```diff
@@ -0,0 +1,66 @@
+body {  
+  width: 100vw;
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 300px;
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-wolox {
+  height: 70px;
+  width: 210px;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+label {
+  height: 22px;
+  color: $black;
+  font-size: 16px;
+  line-height: 24px;
+  position: relative;
+  margin-bottom: 10px;
+}
+
+.data {
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: $white;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.login {
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: $earls-green;
+  position: relative;
+  margin-bottom: 15px;
+  
+}
+
+.border {
```

Comment:
Why a new `div` for this? I think you can achieve the same with just the original element 😄